streamwebs_frontend/streamwebs/tests/models/test_water_quality.py

```diff
@@ -0,0 +1,124 @@
+from django.test import TestCase
+
+from django.contrib.gis.db import models
+from django.apps import apps
+from itertools import chain
+from django.core.exceptions import ValidationError
+
+from streamwebs.models import Site, Water_Quality, validate_wq_site
+
+
+class WaterQualityTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = {
+            'site_name': models.ForeignKey,
+            'date': models.DateField,
+            # 'time': models.TimeField,
+            'school': models.CharField,
+            'teacher': models.CharField,
+            'latitude': models.DecimalField,
+            'longitude': models.DecimalField,
+            'fish_present': models.BooleanField,
+            'live_fish': models.PositiveSmallIntegerField,
+            'dead_fish': models.PositiveSmallIntegerField,
+            'water_temp': models.DecimalField,
+            'air_temp': models.DecimalField,
+            'dissolved_O2': models.DecimalField,
+            'pH': models.DecimalField,
+            'turbidity': models.DecimalField,
+            'salinity': models.DecimalField,
+            'conductivity': models.DecimalField,
+            'total_solids': models.DecimalField,
+            'bod': models.DecimalField,
+            'ammonia': models.DecimalField,
+            'nitrite': models.DecimalField,
+            'nitrate': models.DecimalField,
+            'phosphates': models.DecimalField,
+            'fecal_coliform': models.DecimalField,
+            'notes': models.TextField,
+            'id': models.AutoField
+        }
+
+        # Where does 'Vernier' and 'Manual' fit into the model?
+        # Celcius and Fahrenheit?
+        # Sample number??
```

Comment:
It looks like we'll need a water_temp_unit and air_temp_unit field with a C / F choice. If each measurement needs an associated sample number and vernier/manual flag, that means either a lot of extra fields, or abstracting out a measurements table and making a one-to-many relation between datasheet and measurements (measurement table would have measurement label, sample number, datasheet_id, and unit fields)